extensions/positron-supervisor/package.nls.json

```
@@ -7,6 +7,14 @@
 	"configuration.logLevel.debug.description": "Debug messages",
 	"configuration.logLevel.trace.description": "Verbose tracing messages",
 	"configuration.logLevel.description": "Log level for the kernel supervisor (restart Positron to apply)",
+	"configuration.shutdownTimeout.description": "When should kernels be shut down after Positron is closed?\n\nTimes are in hours, and don't start until the kernel is idle, even after Positron is closed. Restart Positron to apply.",
```

Is it the case that if the user selects an option other than `immediately`, that any in-progress work in the kernel will continue on until it completes, then at that point (when the kernel becomes idle), the timeout begins at that point?

So if we selected a 4 hour timeout and started a computation which ends up taking 2 hours, then the kernel will shutdown in about 6 hours?

A possible wording suggestion if my understanding is accurate!

```suggestion
	"configuration.shutdownTimeout.description": "When should kernels be shut down after Positron is closed?\n\nTimeouts are in hours and start once the kernel is idle. Restart Positron to apply.",
```

---

Yes, that's correct!